CIP-0001/README.md

```diff
@@ -127,7 +127,7 @@ License: CC-BY-4.0
 
 ##### Repository Organization
 
-A CIP must be stored in a specific folder named after its number (4-digit, left-padded with `0`) and in a file called `README.md`. Before a number is assigned, use `????` as a placeholder number (thus naming new folders as `CIP-????`). After a number has been assigned, rename the folder.
+A CIP must be stored in a specific folder named after its number (4-digit, left-padded with `0`) and in a file called `README.md`. Before a number is assigned, use `????` or `XXXX` as a placeholder number (thus naming new folders as `CIP-????` or `CIP-XXXX`). After a number has been assigned, rename the folder.
```

Comment:
```suggestion
A CIP must be stored in a specific folder named after its number (4-digit, left-padded with `0`) and in a file called `README.md`. Before a number is assigned, use `????` or `XXXX` as a placeholder number (thus naming new folders as `CIP-????` or `CIP-XXXX`) or a brief, descriptive text string (e.g. `CIP-my-new-feature`). After a number has been assigned, rename the folder.
```
Several CIPs during the draft process have used descriptive names for the content folder instead of the generic 4-character placeholders `????` and `XXXX`.  This also gives authors the opportunity to name the folder identically with their branch.  Most importantly it gives readers & editors a means to find the added content folder in the PR branch according to some meaningful name.

If there is any doubt about this I'll go back in the commit history for CIPs in which this was used during the editing process & post some links here.  For now I can confirm in my own memory that most times I've suggested this as a third option it's almost always been taken over the 4-character placeholders and sometimes imitated spontaneously by other authors.

Reply:
Yup good spot, I think this is very reasonable.